test/e2e/specs/protractor.conf.js

```diff
@@ -0,0 +1,29 @@
+let SpecReporter = require('jasmine-spec-reporter').SpecReporter;
+
+exports.config = {
+    sauceUser: process.env.SAUCE_USERNAME,
+    sauceKey: process.env.SAUCE_ACCESS_KEY,
```

Comment:
`sauceUser` and `sauceKey` shouldn't be necessary for these tests. These 2 properties are used in travis so that it can create a tunnel to saucelabs.